README.md

```diff
@@ -68,6 +68,12 @@ their impact. We support both Intel and Apple Silicon macs now and contain this
 `ThermalsParser.kt`. This also includes helpful charting APIs for visualizing the data (courtesy
 of our friends at Square).
 
+### Better Properties
+
+Gradle's built-in property support is limited and has surprising behavior, so we have our own 
+system on top of it that has consistent precedence, `local.properties` support, local project 
```

Comment:
Not - just a suggestion with simplified sentence from grammar tool

```suggestion
Gradle's built-in property support is limited and has surprising behavior, so our system has 
consistent precedence, local.properties support, local project properties, ...
```

Reply:
I don't think this wording is clearer tbh. More declarative tone yes, but calling out that we have our own system is sorta the focal point in this one